ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -3564,3 +3564,44 @@ def test_numpy___invert__(
         method_flags=method_flags,
         on_device=on_device,
     )
+    
+# std
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="std",
+    dtype_x_axis=helpers.dtype_values_axis(
```

Comment:
There are assertion errors being thrown due to generated examples, given that the statistical functions have their specific strategy that avoids the overflow errors you can use that🙂
https://github.com/unifyai/ivy/blob/291d9277f4c57a3f3f9e607f5ac4266774f5dcf8/ivy_tests/test_ivy/test_functional/test_core/test_statistical.py#L12

Reply:
Hi @hirwa-nshuti,
I have tried to made some changes in code but they are failing on more tests, so I tried to revert back previous code which has 2 failing test cases but now for that same is also failing on 56 test cases. I don't know what is wrong with this code could u please help me.

Reply:
I messed up with the whole code, sorry for that. But to make it correct I discarded all commits and made new commits, these are the same codes that I was using just before you asked me to remove assertion error and at that time many test cases were passing but now more that half of them are failing, I have tried from my side to fix these but as I already said that I ended up with messing all the code.
So, I request you to please guide me.